packages/uniforms-bootstrap4/src/AutoField.tsx

```diff
@@ -57,5 +57,7 @@ export default function AutoField(originalProps: AutoFieldProps) {
     }
   }
 
-  return createElement(component, originalProps);
+  return 'options' in component && component.options?.kind === 'leaf'
```

Comment:
Actually, why do we need `'options' in component` if we use `component.options?.` operator? Doesn't that mean conditional property access operator checks if `options` exist?

Reply:
It's required to narrow `ComponentType<any> | ReturnType<typeof connectField>` down to `ReturnType<typeof connectField>`.